pkgs/top-level/python-packages.nix

```diff
@@ -490,6 +490,8 @@ in {
 
   azure-multiapi-storage = callPackage ../development/python-modules/azure-multiapi-storage { };
 
+  b4 = callPackage ../applications/version-management/git-and-tools/b4 { };
```

Comment:
python-modules shouldn't contain a `buildPythonApplication`. The package won't appear to downstream dependencies.

If you want to package this a python module, please use buildPythonPackage or `pythonXPackages.toPythonModule`

Reply:
It is a application though, a commandline utility.

Reply:
buildPythonPackage will still show entry points as command line utilities, that difference is in downstream dependencies. Please read https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#building-packages-and-applications